test/index.js

```diff
@@ -3133,10 +3139,10 @@ describe('UnaryType', function() {
 
 describe('BinaryType', function() {
 
-  it('is a quinary function', function() {
+  it('is a sexternary function', function() {
```

Comment:
I was looking around for this term for a while. I gave up and figured you'd set it straight ;)